agnostic-angular/libs/ag/src/lib/menu.component.stories.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  componentWrapperDecorator,
+  moduleMetadata,
+  Story,
+} from '@storybook/angular';
+import { Meta } from '@storybook/angular/types-6-0';
+import { AgModule } from './ag.module';
+import { MenuComponent } from './menu.component';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'AG—Angular (Beta)/Menu',
+  component: MenuComponent,
+  decorators: [
+    // Cannot get preview.js or global decorator solutions to work.
+    // https://storybook.js.org/tutorials/intro-to-storybook/angular/en/composite-component/
+    componentWrapperDecorator(
+      (story) =>
+        `<div style="font-family: var(--agnostic-font-family-body); line-height: 1.4; font-size: var(--agnostic-body);">${story}</div>`
+    ),
+    moduleMetadata({
+      imports: [AgModule],
+    }),
+  ],
+} as Meta;
+
+const actions = {
+  onSelectionChange: action('onSelectionChange'),
+  onTriggerClick: action('onTriggerClick'),
```

Comment:
Longer term I'd like us to implement `onTriggerKeydown` and just listen for case when you press `ArrowDown` and the menu options are closed; this should result in the menu being opened.

Reply:
I think that behavior will make a lot of sense. I'm def on board